src/net.cpp

```diff
@@ -1621,10 +1620,16 @@ void CConnman::ThreadDNSAddressSeed()
         if (HaveNameProxy()) {
             AddOneShot(seed.host);
         } else {
+            if (!seed.supportsServiceBitsFiltering) {
```

Comment:
What if we just always tried the desired subdomain, and fell back to the OneShot on the host if the resolve fails? That way seed operators can update for new flags in the future, and we can get rid of the supportsServiceBitsFiltering logic.

Reply:
Does that not have any annoying interactions with async name lookup changes?

Reply:
Nah, async resolves will just take a functor specifying what to do with the result. In this case, it'll just do ```AddOneShot(seed.host)``` on failure.

Reply:
That would essentially drop @luke-jr DNS seed. @luke-jr: plans to support filtering by service bits soon?

Reply:
Using a DNS Seed as a oneshot still provides at least some useful result, and is what we do for Tor nodes anyway, so I'd somewhat prefer to at least do that. Took @theuni's suggestion here.